Compiler warnings on release branch

I get the following warnings when compiling the deal-9.0 branch with gcc-4.8.4:
```
/home/rengas/Software/dealii/source/lac/constraint_matrix.cc: In member function ‘bool dealii::ConstraintMatrix::is_consistent_in_parallel(const std::vector<dealii::IndexSet>&, const dealii::IndexSet&, MPI_Comm, bool) const’:
/home/rengas/Software/dealii/source/lac/constraint_matrix.cc:113:26: warning: missing initializer for member ‘dealii::ConstraintMatrix::ConstraintLine::index’ [-Wmissing-field-initializers]
   ConstraintLine empty= {};
                          ^
/home/rengas/Software/dealii/source/lac/constraint_matrix.cc:113:26: warning: missing initializer for member ‘dealii::ConstraintMatrix::ConstraintLine::entries’ [-Wmissing-field-initializers]
/home/rengas/Software/dealii/source/lac/constraint_matrix.cc:113:26: warning: missing initializer for member ‘dealii::ConstraintMatrix::ConstraintLine::inhomogeneity’ [-Wmissing-field-initializers]
```
Is this something that is already known? I did not see an open issue or PR for it.

--- a/source/lac/constraint_matrix.cc
+++ b/source/lac/constraint_matrix.cc
@@ -110,7 +110,9 @@ bool ConstraintMatrix::is_consistent_in_parallel(const std::vector<IndexSet> &lo
                                                  const MPI_Comm mpi_communicator,
                                                  const bool verbose) const
 {
-  ConstraintLine empty= {};
+  ConstraintLine empty;
+  empty.inhomogeneity = 0.0;
+
   // Helper to return a reference to the ConstraintLine object that belongs to row @p row.
   // We don't want to make copies but to return a reference, we need an empty object that
   // we store above.

